network-api/networkapi/wagtailpages/customblocks/image_grid.py

```diff
@@ -0,0 +1,28 @@
+from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+
+class ImageGrid(blocks.StructBlock):
+    image = ImageChooserBlock()
+    caption = blocks.CharBlock(
+        required=False,
+        help_text='Please remember to properly attribute any images we use.'
+    )
+    url = blocks.CharBlock(
+        required=False,
+        help_text='Optional URL that this figure should link out to.',
+    )
+    square_image = blocks.BooleanBlock(
+        default=True,
+        required=False,
+        help_text='If left checked, the image will be cropped to be square.'
+    )
+
+
+class ImageGridBlock(blocks.StructBlock):
```

Comment:
let's put this in its own file. It's a small bit of code, but if we're doing this refactor, we might as well do it fully.